src/com/sda/CrazyExperiment.java

```diff
@@ -0,0 +1,8 @@
+package com.sda;
+
+public class CrazyExperiment {
```

Comment:
What about using a different name for this class. I'm not sure "Crazy" is a good term to use. ;-)

Reply:
What about "Cool"?